### DISCUSSION TITLE
Updating from 2.8.0 to 2.12.0 doesn't start

### DISCUSSION THREAD

**Discussion by idemus** [kimai/kimai#4660] [Questions & Answers] [answered]
Hello everyone,
after doing steps explained in 'updating Kimai' (git checkout 2.12.0 -> ... -> bin/console kimai:update) the update process doesn't start because it seems that there is already version 2.12.0 installed.
At the end the success message "Successfully updated to 2.8.0" is displayed.

![image](https://github.com/kimai/kimai/assets/71780381/d7cfd5c7-863a-4742-b4b1-c96b09f84d43)

How can I restart the update process or reset the flag to Kimai 2.8.0?

Best regards
Klaus

**Comment by kevinpapst** [comment, marked as answer]
The screenshots says that you are on 2.8.

I'd wildly guess that you never checked out 2.12 and there is nothing to reset.

What does `git status` tell you?

**Reply by idemus** [reply]
on branch 2.12.0

**Reply by idemus** [reply]
Thanks for the hint, Kevin. I rolled back to commit d8f89d9f3d2e0b20c4ebf102cc7f64f1e4862a4f with
git checkout d8f89d9f3d2e0b20c4ebf102cc7f64f1e4862a4f

and then I could restart the update with stashing my local changes before.
